pages.fr/common/tar.md

```diff
@@ -6,25 +6,29 @@
 
 - Crée une archive à partir de fichiers :
 
-`tar cf {{cible.tar}} {{fichier1 fichier2 fichier3}}`
+`tar cf {{cible.tar}} {{fichier1}} {{fichier2}} {{fichier3}}`
```

Comment:
```suggestion
`tar cf {{chemin/vers/cible.tar}} {{chemin/vers/fichier1 chemin/vers/fichier2 ...}}`
```

Mandatory fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) must be updated): path format is [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths), `tar` doesn't mandate exactly 3 files to be passed.

Reply:
Ah, thank you for clarifying this point, I will keep it in mind for my future contributions.